Studs/Events/Views/EventsViewController.swift

```diff
@@ -8,5 +8,147 @@
+                print("done")
+            }
+        }
+
+        // Set styling
+        view.backgroundColor = UIColor.bgColor
+        eventsTable.backgroundColor = UIColor.bgColor
+
+        // Set up the table view
+        eventsTable.delegate = self
+        eventsTable.dataSource = self
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        if let selectedIndex = eventsTable.indexPathForSelectedRow {
+            eventsTable.deselectRow(at: selectedIndex, animated: true)
+        }
+    }
+
+    // MARK: Table methods
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return sectionTitles.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return data[section].count
+    }
+
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 80
+    }
+
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let header = UIView()
+        let label = UILabel()
+        header.addSubview(label)
+
+        header.backgroundColor = UIColor.bgColor
+        label.text = sectionTitles[section]
+
+        let fontWeight = section == 0 ? UIFont.Weight.heavy : UIFont.Weight.regular
+        label.font = UIFont.systemFont(ofSize: 38, weight: fontWeight)
+        label.bottomAnchor.constraint(greaterThanOrEqualTo: header.bottomAnchor, constant: -10).isActive = true
+        label.leftAnchor.constraint(greaterThanOrEqualTo: header.leftAnchor, constant: 15).isActive = true
+        label.translatesAutoresizingMaskIntoConstraints = false
+
+        return header
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = eventsTable.dequeueReusableCell(withIdentifier: "eventCell", for: indexPath) as? EventTableViewCell else {
+            fatalError("The dequeued cell is not an instance of EventTableViewCell")
+        }
+        let currentData = data[indexPath.section][indexPath.row]
+
+        guard let date = currentData.getDate() else {
+            fatalError("Couldn't format date of event")
+        }
+
+        // Display month and date
+        let dateFormatter = DateFormatter()
+
+        dateFormatter.dateFormat = "MMM"
+        cell.month.text = dateFormatter.string(from: date).uppercased()
+
+        dateFormatter.dateFormat = "d"
+        cell.day.text = dateFormatter.string(from: date)
+
+        cell.companyName.text = currentData.companyName
+
+        return cell
+    }
+
+    // MARK: Navigation
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let selectedEvent = data[indexPath.section][indexPath.row]
+        let storyboard = UIStoryboard(name: "EventsViewController", bundle: nil)
+        guard let eventDetailController = storyboard.instantiateViewController(withIdentifier: "EventDetail")
```

Comment:
Is that preferable? I was thinking about that, but I wasn't sure if it would be better for the two VCs to be in the same story board.

Reply:
I think two view controllers in one storyboard can be fine, but when you start to have more you can get performance problems in Xcode. Otherwise one could think of one storyboard per view controller as a nice architecture.